WcaOnRails/app/controllers/competitions_controller.rb

```diff
@@ -392,12 +392,35 @@ def time_until_competition
     }
   end
 
-  def currency_convert
-    update_rates_if_needed
-    converted = Money.new(params[:value], params[:from_currency]).exchange_to(params[:to_currency])
+  def calculate_dues
+    country_iso2 = Country.where(id: params[:country_id]).pluck(:iso2)[0]
+    country_band = CountryBand.where(iso2: country_iso2).map(&:number)[0]
+
+    update_currency_rates_if_needed
+    currency_subunit = Money::Currency.new(params[:currency_code]).subunit_to_unit
+    money_value = params[:entry_fee].to_i / currency_subunit
+    money_value_usd = Money.new(money_value, params[:currency_code]).exchange_to("USD").cents
+
+    base_fee_usd = money_value_usd * 0.15
```

Comment:
Magic number is bad. Where does it come from? What policy is it based upon? (Those are rhetorical questions).
Please move this to a constant in `CountryBands` (or another model class that you deem more conveninent) and document it thoroughly!

Reply:
added to `CountryBands` for convenience, but I'm thinking if this is bad practice because the constant doesn't really have much to do with countries or bands.

Reply:
It's "less bad" practice than having the number `0.15` randomly appear as integer literal in your code